e2e-tests/01-setup.yml

```diff
@@ -0,0 +1,36 @@
+name: Policy Server end-to-end setup
+vars:
+  pidfile: policy-server.pid
+
+testcases:
+- name: Build policy-server binary with release mode. Can take quite some time...
+  steps:
+  - script: cargo build --release
+    type: exec
```

Comment:
Why do we need to build it here if the makefile already does that? Is the makefile dependency just to check if the code is building?

Reply:
Building in release mode takes a long time. I prefer to have a dedicated step inside of the GH action because that provides a better feedback.